web/src/scroll_bar.ts

```diff
@@ -10,6 +10,29 @@ export function set_layout_width(): void {
     }
 }
 
+export function handle_overlay_scrollbars(): void {
+    // If right sidebar scrollbar overlaps with browser scrollbar, move the right
+    // sidebar scrollbar to the left. Done on fluid screen width and when scrollbars overlap.
+    const scrollbar_width = window.innerWidth - document.documentElement.clientWidth;
+    if (scrollbar_width === 0) {
```

Comment:
Maybe just return when `scrollbar_width !== 0`? I think stylistically that's preferable to a function-long `if` statement with all the logic tucked inside.

Reply:
We need to remove `has-overlay-scrollbar` in that case, so I think the early return pattern wouldn't be correct without additional changes.

Reply:
Yeah.